lib/streamlit/server/Server.py

```diff
@@ -372,6 +382,7 @@ def _send_message(self, ws, session_info, msg):
         ws.write_message(serialize_forward_msg(msg_to_send), binary=True)
 
     def stop(self):
+        LOGGER.info(" Tearing down...")
```

Comment:
Don't use `LOGGER.info` for this, since it appears every time the user shuts down the server. Instead, use `click` and make it pretty with colors:

```
click.secho("  Stopping...", fg="blue")
```

(I also think "Stopping..." is nicer than "Tearing down..." :wink: )